models/channel/src/channel-draft/presets/index.ts

```diff
@@ -1,10 +1,12 @@
+import * as SampleDataFashion from './sample-data-fashion';
 import withInventorySupplyAndProductDistributionRoles from './with-inventory-supply-and-product-distribution-roles';
 import withInventorySupplyAndProductDistributionRolesNoAddress from './with-inventory-supply-and-product-distribution-roles-no-address';
 import withInventorySupplyRole from './with-inventory-supply-role';
 import withInventorySupplyRoleNoAddress from './with-inventory-supply-role-no-address';
 import withProductDistributionRole from './with-product-distribution-role';
 
 const presets = {
```

Comment:
Do we need to update the `channel/index.ts` file to include exports of the draft presets? Without it, what would the import look like?

Reply:
Addressed in 0850dc85538824e43a82848df7469440000c7a33